images/scripts/debian.setup

```diff
@@ -149,10 +149,15 @@ echo 'MODULES=dep' > /etc/initramfs-tools/conf.d/modules-dep.conf
 
 if grep -q 'ID=debian' /etc/os-release; then
     # For debugging libvirt crashes
-    DEBUG_PACKAGES="\
-libvirt0-dbgsym  \
-libglib2.0-0-dbgsym \
-"
+    DEBUG_PACKAGES="libvirt0-dbgsym"
+
+    if grep -q 'VERSION_ID="12"' /etc/os-release; then
+        # Debian 12 (Bookworm) has no separate -dbg packages
```

Comment:
This comment is confusing now. I'm not even sure how it got in there. But I'll let the tests finish first.